vendor/bat-native-ads/src/bat/ads/internal/ml/pipeline/text_processing/embedding_processing.h

```diff
@@ -0,0 +1,58 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_ML_PIPELINE_TEXT_PROCESSING_EMBEDDING_PROCESSING_H_
+#define BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_ML_PIPELINE_TEXT_PROCESSING_EMBEDDING_PROCESSING_H_
+
+#include <cstdint>
+#include <map>
+#include <memory>
+#include <string>
+
+#include "bat/ads/internal/ml/data/vector_data.h"
```

Comment:
not needed here, already imported in `.cc`

Reply:
Removed